src/block-components/progress-bar/edit.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * External dependencies
+ */
+import { unescape, debounce } from 'lodash'
+import { i18n } from 'stackable'
+import {
+	PanelAdvancedSettings,
+	InspectorStyleControls,
+	AdvancedToggleControl,
+	AdvancedRangeControl,
+	AdvancedTextControl,
+	ColorPaletteControl,
+} from '~stackable/components'
+import {
+	useBlockAttributesContext, useBlockSetAttributesContext, useAttributeEditHandlers,
+} from '~stackable/hooks'
+import { DEFAULT_PERCENT, DEFAULT_THICKNESS } from '~stackable/block-components/progress-bar/attributes'
```

Comment:
Let's import this internally so we won't have to run through aliasing.
```suggestion
import { DEFAULT_PERCENT, DEFAULT_THICKNESS } from './attributes'
```